src/streamlink/plugins/vimeo.py

```diff
@@ -25,6 +25,22 @@
     r"https?://(player\.vimeo\.com/video/\d+|(www\.)?vimeo\.com/.+)",
 ))
 class Vimeo(Plugin):
+    VIEWER_URL = "https://vimeo.com/_next/viewer"
+    OEMBED_URL = "https://vimeo.com/api/oembed.json"
+    API_URL = "https://{0}/{1}?fields=config_url"
+    _uri_schema = validate.Schema(
+        validate.parse_json(),
+        validate.all({
+            "uri": str,
+        }),
+    )
+    _viewer_schema = validate.Schema(
+        validate.parse_json(),
+        validate.all({
+            "jwt": str,
+            "apiUrl": str,
+        }),
+    )
```

Comment:
`validate.all` is not needed here. It's a logical conjunction of its input arguments, and you only define one, a dict instance.

Please also don't define more schemas as class attributes. This is a bad pattern that should be avoided, as it leads to a lot of unnecessary initialization when loading the plugin file. Initialize the schemas before or when making the HTTP request.